solidity/dashboard/src/services/tbtc-authorization.service.js

```diff
@@ -202,23 +202,26 @@ const fetchBondingData = async (web3Context) => {
       )
     }
 
-    for (let i = 0; i < operators.length; i++) {
+    for (const [
+      operatorAddress,
+      isAddressWithdrawable,
+    ] of operators.entries()) {
       const delegatedTokens = await fetchDelegationInfo(
         web3Context,
-        operators[i][0]
+        operatorAddress
       )
       const availableEth = await fetchAvailableAmount(
         web3Context,
-        operators[i][0]
+        operatorAddress
       )
 
-      const bondedEth = operatorBondingDataMap.get(operators[i][0])
-        ? operatorBondingDataMap.get(operators[i][0])
+      const bondedEth = operatorBondingDataMap.get(operatorAddress)
+        ? operatorBondingDataMap.get(operatorAddress)
         : 0
 
       const bonding = {
-        operatorAddress: operators[i][0],
-        isWithdrawable: operators[i][1],
+        operatorAddress: operatorAddress,
```

Comment:
I think we can just `operatorAddress,` instead of `operatorAddress: operatorAddress`

Reply:
nice, simplified!